Orange/widgets/visualize/utils/heatmap.py

```diff
@@ -512,7 +512,7 @@ def setHeatmaps(self, parts: 'Parts') -> None:
                 items=labels, parent=self,
                 alignment=Qt.AlignLeft | Qt.AlignVCenter,
                 orientation=Qt.Horizontal,
-                autoScale=True,
+                autoScale=False,
```

Comment:
This would leave the annotations unscaled and cramped when there are more columns

![Screen Shot 2020-05-15 at 11 52 35](https://user-images.githubusercontent.com/4716745/82037729-d758a280-96a2-11ea-8f32-7bc143f74e61.png)

Reply:
Yeah, I thought it cannot be that easy. I only happens for a single variable. If len(var) == 1? And have two different parameters, depending on the number of variables?